cli/ops/os.rs

```diff
@@ -148,3 +149,16 @@ fn op_os_release(
   let release = sys_info::os_release().unwrap_or_else(|_| "".to_string());
   Ok(JsonOp::Sync(json!(release)))
 }
+
+fn op_freemem(
+  state: &State,
+  _args: Value,
+  _zero_copy: &mut [ZeroCopyBuf],
+) -> Result<JsonOp, OpError> {
+  state.check_unstable("Deno.freemem");
+  state.check_env()?;
+  match sys_info::mem_info() {
+    Ok(info) => Ok(JsonOp::Sync(json!(info.free))),
```

Comment:
Maybe this op should be called mem_info instead and return the whole `sys_info::MemInfo` to JavaScript?

Reply:
It make sense. I made it return only _free memory_ because in issue https://github.com/denoland/deno/issues/3802 it is related to Node.js `os.freemem()` function.

Reply:
That could be added later in std's node compat.

Reply:
Sure, now that `memoryInfo()` returns all memory info the `freemen()` function can be something like this:
```ts
function freemen(): number {
    return memoryInfo().free;
}
```
If you think it could be fine I can add it.